Sources/MongoSwift/BSON/Document+Sequence.swift

```diff
@@ -242,6 +242,19 @@ public class DocumentIterator: IteratorProtocol {
         return values
     }
 
+    /// Returns the current value, equivalently to the `currentValue` property, but in the case of error, will throw
```

Comment:
Hm, I suppose you might be right. At the same time, `currentValue` is a computed property, so is it actually a verb? I think computed properties actually blur the line here.

In either case though, I wrote "equivalently" here because it is referring to the action of returning ("Returns ... , equivalently ...").

What do you think about this?

Reply:
How about:

> Returns the current value (equivalent to the `currentValue` property) or throws on error

I don't think we need to say "instead of causing a fatal `preconditionFailure`". This is internal and we probably already know that subscript has no choice but to crash, while we can throw from this method.

Reply:
Awesome! Sounds good. Done.